src/main/java/net/william278/velocitab/packet/ScoreboardManager.java

```diff
@@ -211,18 +214,16 @@ public void resendAllTeams(@NotNull Player player) {
             if (roles.contains(role)) {
                 return;
             }
-
             roles.add(role);
 
-            final String nametag = nameTags.getOrDefault(role, "");
-            if (nametag.isEmpty()) {
-                return;
+            // Send packet
+            if (nameTags.containsKey(role)) {
```

Comment:
Maybe we could use an Optional#ofNullable and a getOrDefault with null. It's impossible to have a key with a null value in this case. Removing the containsKey would improve performance. Maybe we could put that in a method because it would be called from multiple points in the code.

Reply:
uhh maybe yeah not sure what's faster in this case.